data/data/agent/files/usr/local/bin/get-container-images.sh.template

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# shellcheck disable=SC1091
+. /usr/local/bin/release-image.sh
+
+ImagesEnvFile=${1}/agent-images.env
```

Comment:
tbh I'd sooner hard-code `/var/run` here than have to go looking for the place where this is called to figure out where the config ends up.

Reply:
Changed